gensim/models/hdpmodel.py

```diff
@@ -507,9 +508,13 @@ def save_options(self):
             fout.write('eta: %s\n' % str(self.m_eta))
             fout.write('gamma: %s\n' % str(self.m_gamma))
 
-    def hdp_to_lda(self):
+    def hdp_to_lda(self, model=False):
```

Comment:
I do agree about the parameters of number of iterations and gamma, but I couldn't find an equivalent of that in `hdp`. Any ideas or suggestions?